roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1066,36 +1066,40 @@ def get_openshift_version(facts, cli_image=None):
         if 'version' in facts['common'] and facts['common']['version'] is not None:
             return facts['common']['version']
 
-    if os.path.isfile('/usr/bin/openshift'):
-        _, output, _ = module.run_command(['/usr/bin/openshift', 'version'])
-        version = parse_openshift_version(output)
-
+    # If this is a containerized host, we need to get the openshift version
```

Comment:
I'm sure there are still problems on my branch related to docker not being installed.  I think we'll need the refactorings in my PR however late yesterday I noticed they break the 3.2 upgrade fairly severely because of how we're calculating the version of docker to install.